### ISSUE TITLE
Setup für Erfüllung betrieblicher Pflichten

### ISSUE THREAD

**Issue by Highkovic** [ImmunHelden/ImmunHelden.de#64]
Für den Betrieb unserer GbR müssen wir nach Einreichung des Antrags folgende Punkte erledigen:

GbR-Konto errichten (Vorschlag: €​ Penta​ - Business-Tarif “Basic”: https://getpenta.com/de/ - kostenfrei bis 50 Buchungen im Monat - für GbRs geeignet)
Account bei Elster anlegen für Umsatzsteuer- und Gewerbesteuerabwicklung
Prototypefund Finanz und Steuerinfo Broschüre_ Stand 10.2.2020.pdf

**Comment by weliveindetail** [comment]
Unsere Einkünfte sind befreit von Umsatzsteuer- und Gewerbesteuer. Für ein Konto mit positivem Saldo können wir meines Wissens nach eins von unseren angeben. Siehe https://github.com/ImmunHelden/ImmunHelden.de/issues/39#issuecomment-621971803

Ist hier dann noch etwas zu tun?

**Comment by Highkovic** [comment]
Genau, sie sind befreit, das passiert aber nicht automatisch, wir müssen das vorstrecken und bekommen es erstattet, wenn ich das richtig verstanden habe. Auf jeden Fall müssen wir eine Meldung vornehmen. Da ist also noch was zu tun (Anmeldung bei Elster Online), doch das machen wir als nächsten Schritt nach der Gewerbeanmeldung--Diese Nachricht wurde von meinem Android Mobiltelefon mit WEB.DE Mail gesendet.Am 07.05.20, 13:08 schrieb "Stefan Gränitz" <notifications@github.com>:

   
  Unsere Einkünfte sind befreit von Umsatzsteuer- und Gewerbesteuer. Für ein Konto mit positivem Saldo können wir meines Wissens nach eins von unseren angeben. Siehe #39 (comment) 
  Ist hier dann noch etwas zu tun?

**Comment by weliveindetail** [comment]
> Genau, sie sind befreit, das passiert aber nicht automatisch, wir müssen das vorstrecken und bekommen es erstattet, wenn ich das richtig verstanden habe.

Hast du das im Chat von Prototype überprüft? Von Elster hat in der Q&A mit dem DLR niemand gesprochen. Auch die Anmeldung als Gewerbe, woher kommt die Info dass wir das brauchen? Die Auszahlung geht doch auf unsere eigenen Konten. Steuerlich müssen wir das ggf. selbst geltend machen. Die GbR bildet doch eigentlich nur den rechtlichen Rahmen.

**Comment by Highkovic** [comment]
Steht in einem Dokument, das sie in Chat zur Verfügung gestellt haben. Da sind steuerliche Fragen beantwortet. Habe das Dokument auch hier im Issue hochgeladen. Die Frage nach der Gewerbeanmeldung hat ein anderer Nutzer bereits gestellt: ihm wurde wohl gesagt (nicht öffentlich), das wir das bräuchten, auch wenn sein Steuerberater anderen Meinung ist (weil fehlende Gewinnerzielungsabsicht). Ich kann das aber dennoch auch nochmals doppelt absichern.

   
   
   Genau, sie sind befreit, das passiert aber nicht automatisch, wir müssen das vorstrecken und bekommen es erstattet, wenn ich das richtig verstanden habe. 
   
  Hast du das im Chat von Prototype überprüft? Von Elster hat in der Q&A mit dem DLR niemand gesprochen. Auch die Anmeldung als Gewerbe, woher kommt die Info dass wir das brauchen? Die Auszahlung geht doch auch auf unsere eigenen Konten. Die GbR bildet doch eigentlich nur den rechtlichen Rahmen.

**Comment by weliveindetail** [comment]
Auf die Agenda, falls noch relevant

**Comment by Highkovic** [comment]
Hi zusammen,

andere Gruppen habe das Thema Gewerbeanmeldung bereits verifiziert, das hängt wohl ein wenig vom Finanzamt ab. grundsätzlich scheint es aber nicht notwendig zu sein, wenn wir folgende Schritte befolgen:
"Anschreiben von wegen nur Zuwendung und umsatzsteuerbefreit. Falls dann, wie bei uns, trotzdem eine Aufforderung zu Umsatzsteuervoranmeldung kommt, anrufen und nochmal erklären. Dann muss nur eine Umsatzmitteilung zum Projektende eingereicht werden über die gesamt Fördersumme."
